src/views/mainLayout.js

```diff
@@ -14,6 +14,27 @@ const mainLayout = data => {
   const metaDescription = data.meta?.socialDescription ?? getMessage('meta-desc-2')
   const pageUrl = `${AppConstants.SERVER_URL}${data.pathname ?? '/'}`
 
+  const shouldShowRecruitmentBanner = () => {
+    // Only show if ENVs are set and user language is set to "en"
+    if (
+      AppConstants.RECRUITMENT_BANNER_TEXT &&
+      AppConstants.RECRUITMENT_BANNER_LINK &&
+      getLocale().includes('en')
```

Comment:
question: Is this the best way to test for English?

Reply:
Q: What/who is "english"?
I'm in Canada these days, is the survey available to me? Is that $100 Canadian dollars, or US dollars? Would it also apply to UK "en"? Do they get paid in pounds or quids or something?